psi/bld/prepare-PSI.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+#version=1.0
+
+ 
+function PSIcreation {  
+  
+    echo "Called function to create PSI in our script."  
```

Comment:
Sorry I didn't start review the PR since this looks like dummy function for now. Are we planning to implement this function in future PRs? or will improve with this PR?

Reply:
We have whole repository in our environment which is building PSI for ZOWE, and we want to somehow incorporate it to zowe environment, namely this sript https://github.gwd.broadcom.net/MFD/PSI-automation/blob/master/ZOWE/create.sh
![image](https://user-images.githubusercontent.com/45940302/129928692-48cb46da-d897-4dfa-b6dc-67ec9ff154e3.png)